opencontrail/control.sls

```diff
@@ -94,15 +94,16 @@ opencontrail_control_entrypoint:
 {%- endif %}
 
 {%- if common.distribution == "juniper" %}
-/etc/contrail/supervisord_config_files/ifmap.ini:
+add_etc_contrail_supervisord_config_files_ifmap.ini:
```

Comment:
What a horrible resource name is that?
Use opencontrail_supervisord_ifmap or something more meaningful